### PR TITLE
fix(ui-shell): add aria label to sidenav with complimentary landmark

### PR DESCRIPTION
**Pull request by dakahn**
Elements having a WAI-ARIA complementary role such as `<aside>` used in our sidenav must have a label specified with aria-label.

This PR gives the `<aside>` element in the sidenav an `aria-label="side navigation"` silencing the error thrown by our automated test and providing a more adequate description for our users.

### CODE REVIEW

File: src/components/UIShell/SideNav.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ export default class SideNav extends React.Component {
     });
 
     return (
-      <aside className={className}>
+      <aside aria-label="side navigation" className={className}>
```
Comment (asudoh) [inline src/components/UIShell/SideNav.js]: Should this be translatable?

Comment (dakahn) [reply src/components/UIShell/SideNav.js]: Oh, yeah probably! I'll investigate how to make that happen.

Comment (joshblack) [reply src/components/UIShell/SideNav.js]: Should this use what's in `accessibilityLabel`?

**Comment by dakahn** [comment]
I I pivoted with my approach and after some research decided to nix the wrapping `<aside>` element altogether. This should simplify things. I moved the wrapping `className` attribute down to the formerly descendant `<nav>` element which seemed to work fine in the storybook.

That said, I'd give this one a close inspection just to make sure I'm not inadvertently breaking something.

**Pull request merged**